tests/fcm-push-service.test.ts

```
@@ -0,0 +1,56 @@
+const sendMock = mock(() => Promise.resolve("ignored"));
+
+void mock.module("firebase-admin/messaging", () => ({
+  getMessaging: () => ({ send: sendMock }),
+}));
+void mock.module("@/utils/firebase", () => ({
+  getFirebaseApp: () => ({}),
+}));
+
+const { FcmPushService } = await import(
+  "@/api/v2/notifications/fcm-push.service"
+);
```

ESLint

[error] Replace `·await·import(⏎··"`@/api/v2/notifications/fcm-push.service`"⏎` with `⏎··await·import("`@/api/v2/notifications/fcm-push.service`"`

(prettier/prettier)